tests/qam-minimal/install_patterns.pm

```diff
@@ -25,14 +25,27 @@ use utils;
 use qam;
 use testapi;
 
+sub install_packages {
+    my $patch_info = shift;
+    my $pattern    = qr/\s+(.+)(?!\.src)\..*\s<\s.*/;
+
+    # loop over packages in patchinfo and try instalation
+    foreach my $line (split(/\n/, $patch_info)) {
+        if (my ($package) = $line =~ $pattern) {
+            script_run("zypper -n in $package", 700);
+            save_screenshot;
+        }
+    }
+}
+
 sub run {
     my ($self) = @_;
+    my $patch = get_var('INCIDENT_PATCH');
```

Comment:
before declaring this to the `patch` variable, it would be a good practice to make sure that the env variable `INCIDENT_PATCH` exists and it's not empty. e.g.


```
# Check if INCIDENT_PATCH variable exists and has a non-empty value 
die "openQA Error -> Cannot find the maintenance incident patch" unless get_var('INCIDENT_PATCH');
```